$NetBSD: patch-af,v 1.5 2006/07/12 15:38:29 rillig Exp $

Fix gcc4 error:
error: array type has incomplete element type

--- mozilla/security/nss/lib/pki1/oiddata.h.orig	2002-01-04 05:22:07.000000000 +0000
+++ mozilla/security/nss/lib/pki1/oiddata.h
@@ -43,6 +43,8 @@ static const char OIDDATA_CVS_ID[] = "@(
 #include "nsspki1t.h"
 #endif /* NSSPKI1T_H */
 
+#include "pki1t.h"
+
 extern const NSSOID nss_builtin_oids[];
 extern const PRUint32 nss_builtin_oid_count;
 
